webpack.config.ts

```diff
@@ -65,9 +65,9 @@ const config: Configuration = {
       },
       {
         test: /\.(png|jpg|jpeg|gif|svg|woff2?|ttf|eot|otf)(\?.*$|$)/,
-        type: 'asset/resource',
-        generator: {
-          filename: 'assets/[name].[ext]',
+        loader: 'file-loader',
```

Comment:
Yi, can you explain this change? This is from the template I believe.

Reply:
This is for the argo icon image. The previous setting didn't work for the argo cd icon png file. I copied this from `console/frontend/webpack.confg.ts` which is working for the static plugin.

Reply:
Ok, makes sense. We're trying to load the icon.